src/constructs/autoscaling/asg.test.ts

```diff
@@ -133,22 +128,22 @@ describe("The GuAutoScalingGroup", () => {
       additionalSecurityGroups: [securityGroup, securityGroup1, securityGroup2],
     });
 
-    expect(stack).toHaveResource("AWS::AutoScaling::LaunchConfiguration", {
+    Template.fromStack(stack).hasResourceProperties("AWS::AutoScaling::LaunchConfiguration", {
       SecurityGroups: [
         {
-          "Fn::GetAtt": [`GuHttpsEgressSecurityGroup${app}89CDDA4B`, "GroupId"],
+          "Fn::GetAtt": [Match.stringLikeRegexp(`GuHttpsEgressSecurityGroup${app}*`), "GroupId"],
         },
         {
           "Fn::GetAtt": ["WazuhSecurityGroup", "GroupId"],
         },
         {
-          "Fn::GetAtt": ["SecurityGroupTestingA32D34F9", "GroupId"], // auto-generated logicalId
+          "Fn::GetAtt": [Match.stringLikeRegexp("SecurityGroupTesting"), "GroupId"],
         },
         {
-          "Fn::GetAtt": ["SecurityGroup1TestingCA3A17A4", "GroupId"], // auto-generated logicalId
+          "Fn::GetAtt": [Match.stringLikeRegexp("SecurityGroup1Testing"), "GroupId"],
         },
         {
-          "Fn::GetAtt": ["SecurityGroup2Testing6436C75B", "GroupId"], // auto-generated logicalId
+          "Fn::GetAtt": [Match.stringLikeRegexp("SecurityGroup2Testing"), "GroupId"],
```

Comment:
This is a lot nicer, IMO, as we're not matching the exact GUID, thus it becomes more flexible.